src/Frontend.hs

```diff
@@ -325,7 +325,8 @@ aulaSpace space
 
 type AulaUser =
        "ideas"       :> GetH (Frame PageUserProfileCreatedIdeas)
-  :<|> "delegations" :> GetH (Frame PageUserProfileDelegatedVotes)
+  :<|> "delegations" :> "global" :> GetH (Frame PageUserProfileDelegatedVotes)
+  :<|> "delegations" :> "class"  :> GetH (Frame PageUserProfileDelegatedVotes)
```

Comment:
it would be nice to have the class here explicitly, as it won't be uniquely identified by the uid any more soon.  but since this implies changes to the UI, i'm ok with postponing the change here, too.